lace/emulator/gp_emulator.py

```diff
@@ -19,57 +20,91 @@ class GPEmulator:
     """
     def __init__(self,basedir="/lace/emulator/sim_suites/Australia20/",
                 p1d_label=None,skewers_label=None,
-                max_archive_size=None,verbose=False,kmax_Mpc=10.0,
-                paramList=None,train=True,drop_tau_rescalings=True,
-                drop_temp_rescalings=True,keep_every_other_rescaling=False,
-                undersample_z=1,emu_type="k_bin",z_max=5,z_list=None,
-                passarchive=None,set_noise_var=1e-3,asymmetric_kernel=True,
-                check_hull=False,set_hyperparams=None,
-                paramLimits=None,rbf_only=True,
+                verbose=False,kmax_Mpc=4.0,
+                paramList=None,train=True,
+		drop_tau_rescalings=True,
+                drop_temp_rescalings=True,
+                emu_type="polyfit",
+		z_max=5,
+                passarchive=None,
+		set_noise_var=1e-3,
+		asymmetric_kernel=True,
+                check_hull=False,
+		set_hyperparams=None,
+                paramLimits=None,
+		rbf_only=True,
                 emu_per_k=False,
-                reduce_var_k=False,
-                reduce_var_z=False,
-                reduce_var_mf=False):
+		ndeg=5,
```

Comment:
This is another important change in the default... if we make all these changes, we should make sure to clearly explain in the README how to reproduce the emulator from Pedersen et al (2021, k_bin) and from Pedersen et al. (2023, polyfit with ndeg=4).